app/src/main/java/com/hbt/demoapplication/MainActivity.kt

```diff
@@ -0,0 +1,45 @@
+package com.hbt.demoapplication
+
+import android.icu.number.NumberFormatter
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/luudan0079/DemoRepo/blob/b15c3fab4ec256000f4393ef2b47eb7d1ac473e0/app/src/main/java/com/hbt/demoapplication/MainActivity.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>